teams/wg-gamedev.toml

```diff
@@ -0,0 +1,19 @@
+name = "wg-gamedev"
+wg = true
+
+[people]
+leads = ["alexene"]
```

Comment:
Please capitalize your GitHub username (as it's shown in the UI).

Reply:
Ok, thanks, I thought it's the alexene (as in the .toml) file.

Reply:
The file name also needs to be changed. This should have been caught by `check` but username validation is not implemented yet.

Reply:
Yup, I managed to eventually do it, thanks.